SEC/Equities/EquityInstruments.rdf

```diff
@@ -406,12 +420,73 @@
 		<skos:definition xml:lang="en">position in an equity instrument</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-eq-eq;EquityRedemptionProvision">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;RedemptionProvision"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasMinimumRedemptionPrice"/>
+				<owl:onClass rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasRedemptionPremium"/>
+				<owl:onClass rdf:resource="&fibo-fnd-acc-cur;MonetaryPrice"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;hasEarliestRedemptionDate"/>
+				<owl:onClass rdf:resource="&fibo-fnd-dt-fd;ExplicitDate"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;isRedeemableAtIssuerOption"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&xsd;boolean"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-eq-eq;isRedeemableAtShareholderOption"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
```

Comment:
Can both this and IssuerOption be false? True? Seems like an enum would be tighter than 2 booleans.

Reply:
Probably not, but how important is it to exclude the case where they are both false?

Reply:
Not hugely important but it would seem a neater model to have a en enumeration "RedemptionInitiation" or similar. I'm not generally fan of booleans, since it makes it harder to document the meaning of each case.

Reply:
BTW if it's at ShareholderOption is that the same as a RetractablePreferredShare https://pocketsense.com/redeemable-vs-retractable-preferred-shares-3877.html ?
We should have the synonym.

Reply:
Raised issue #1136